nengo/builder/connection.py

```diff
@@ -54,6 +65,18 @@ def build_linear_system(model, conn, rng):
     return eval_points, activities, targets
 
 
+def multiply(x, y):
```

Comment:
Nit means an optional nitpick, so you don't have to change it. But, yes, I personally think other helper functions in this file should have docstrings. It takes 5 seconds to write the one-liner, and can significantly aid in the reuse of code and in quickly understanding the scope of helpers.